<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Eugenya-ПК" ignored="4" tests="3" failures="1" timestamp="06 янв 2018 14:10:52 GMT" time="2.814" errors="0">
  <testcase name="createNewBoardTest" time="1.51" classname="Tests">
    <failure type="java.lang.AssertionError" message="2 expectations failed.
Expected status code &amp;lt;200&amp;gt; but was &amp;lt;400&amp;gt;.

Expected content-type &amp;quot;JSON&amp;quot; doesn&amp;apos;t match actual content-type &amp;quot;text/plain; charset=utf-8&amp;quot;.
">
      <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <200> but was <400>.

Expected content-type "JSON" doesn't match actual content-type "text/plain; charset=utf-8".

	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:471)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$3.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:88)
	at io.restassured.internal.ValidatableResponseOptionsImpl.specification(ValidatableResponseOptionsImpl.java:311)
	at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:292)
	at Tests.createNewBoardTest(Tests.java:83)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- createNewBoardTest -->
  <testcase name="getBoardById" time="1.892" classname="Tests"/>
  <testcase name="getBoardCardsList" time="1.892" classname="Tests"/>
  <testcase name="getAllUserBoards" time="0.0" classname="Tests">
    <ignored/>
  </testcase> <!-- getAllUserBoards -->
  <testcase name="getCardByShortId" time="0.0" classname="Tests">
    <ignored/>
  </testcase> <!-- getCardByShortId -->
  <testcase name="postNewCommentToCard" time="0.0" classname="Tests">
    <ignored/>
  </testcase> <!-- postNewCommentToCard -->
  <testcase name="getCardByUniqueId" time="0.0" classname="Tests">
    <ignored/>
  </testcase> <!-- getCardByUniqueId -->
</testsuite>
